docs/source/keywords.rst

```diff
@@ -160,6 +160,10 @@ General options
       * Aggressively vectorize loops
       * ``OFF``
 
+    * * ``Kokkos_ENABLE_COMPILE_AS_CMAKE_LANGUAGE``
+      * Analogous to CUDA/HIP as CMake language
```

Comment:
The doc string in `CMake` is
```suggestion
      * Whether to use native cmake language support
```

Reply:
Sounds good, but I deviated somewhat from your suggestion.  The "use native CMake ..." is the more general, so I put it ahead of the other statement.